.dockerignore

```diff
@@ -0,0 +1 @@
+.git
```

Comment:
Ignoring .git breaks ursula's attempts at figuring out the name of the instances to boot up. The ursula/ directory gets put into the image, but the .git dir doesn't come with it, so attempts to read the tree for git information fails.

```
?fix_dockerfile_for_jason ~/src/ursula> docker run 82b4ec66440a
fatal: Not a git repository (or any of the parent directories): .git
fatal: Not a git repository (or any of the parent directories): .git
fatal: Not a git repository (or any of the parent directories): .git
fatal: Not a git repository (or any of the parent directories): .git
[ERROR] /root/.stackrc must be present
```